core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -258,30 +260,93 @@ var TopicEditorPage = function() {
       'Create subtopic modal taking too long to disappear.');
   };
 
-  this.dragSkillToSubtopic = async function(skillIndex) {
-    var target = element(by.css('.protractor-test-subtopic-column'));
-    await waitFor.visibilityOf(target,
-      "Subtopic's skills list taking too long to appear.");
-    var toMove = await skillCards.get(skillIndex);
+  this.dragSkillToSubtopic = async function(skillDescription, subtopicIndex) {
+    await waitFor.visibilityOf(await uncategorizedSkills.first(),
+      'Uncategorized skills taking too long to appear.');
+    const target = (subtopicColumns.get(subtopicIndex));
+    var uncategorizedSkillIndex = -1;
+    for (var i = 0;i < await uncategorizedSkills.count();i++) {
+      if (skillDescription === await uncategorizedSkills.get(i).getText()) {
+        uncategorizedSkillIndex = i;
+        break;
+      }
+    }
+    expect(uncategorizedSkillIndex).not.toEqual(-1);
+    var toMove = await uncategorizedSkills.get(uncategorizedSkillIndex);
     await dragAndDrop(toMove, target);
   };
 
-  this.dragSkillBetweenSubtopics = async function(
-      fromSubtopicIndex, skillCardIndex, toSubtopicIndex) {
-    var subtopicCol = await subtopicColumns.get(fromSubtopicIndex);
-    skillNamesElems = subtopicCol.all(
-      by.css('.protractor-test-assigned-skill-card-text'));
-    var toMove = await skillNamesElems.get(skillCardIndex);
-    var target = await subtopicColumns.get(toSubtopicIndex);
-    await dragAndDrop(toMove, target);
+  this.navigateToReassignModal = async function() {
+    var reassignSkillButton = element(
+      by.css('.protractor-test-reassign-skill-button'));
```

Comment:
Can you move this to the top of the file with the other selectors?

Reply:
Done.